target/linux/ramips/image/mt7621.mk

```diff
@@ -719,6 +724,25 @@ define Device/lenovo_newifi-d1
 endef
 TARGET_DEVICES += lenovo_newifi-d1
 
+define Device/linksys_e5600
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 26624k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E5600
+  DEVICE_PACKAGES := kmod-usb3 kmod-mt76 kmod-mt7615e \
```

Comment:
[```kmod-mt76```](https://github.com/openwrt/openwrt/blob/e3ab355ddeb302ce0c002badf7f62bf1c6f7ab69/package/kernel/mt76/Makefile#L45-L50) is a meta package for mt76x2/mt7603, so please use ```kmod-mt7603``` instead for MT7603 chip only.